Automattic/jetpack
# Writing prompts block: load prompts when user not connected to WP.com

## Proposed changes:

Fixes wpcom/v3/blogging-prompts endpoint so that users not connected to WP.com can still retrieve prompts.

### Other information:


## Jetpack product discussion

PT: pe7F0s-uo-p2

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* Sandbox public-api.wordpress.com and apply the changes from this PR to your sandbox (`bin/jetpack-downloader test jetpack fix/blogging-prompt-wo-wpcom-user`)
* Make sure your Jetpack site is connected to WP.com and beta blocks are enabled (`define( 'JETPACK_BETA_BLOCKS', true );`)
* Created a new editor user on your site and login as that user.
* Create a new post and insert a writing prompt block. The prompt should load without error.
